{% from "macros.html" import create_alert_msg with context %}
{% from "macros.html" import create_navbar with context %}

{% set args=["js/main"] %}

{% extends 'base.html' %}   
 
{% block content %}
<nav class="navbar navbar-expand bg-primary navbar-dark static-top px-2">
    <a class="navbar-brand mr-1" href="{{ url_for('main.home') }}">
        <span class="ms-3">{{display_name}}</span>
    </a>
</nav>
<div id="wrapper">
    <div id="content-wrapper">
        <div class="container">
            <div class="row">
                <div class="col-12 m-auto text-center">   
                    {{create_alert_msg()}}                  
                    <h1 class="display-1">500</h1>
                    <p class="lead">Siamo spiacenti, sembra che si sia verificato un errore interno del server.</br> Puoi <a class="text-primary" href="javascript:history.back()">tornare indietro </a>alla pagina precedente, 
                        oppure <a class="text-primary" href="{{ url_for('main.home') }}">ritornare al Cruscotto</a>.</p>  
                </div>
            </div>
        </div>
    </div>
</div>
{% endblock %}